StripeConnect/StripeConnect/Source/Internal/Webview/MessageSenders/ReturnedFromSecureWebViewSender.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  ReturnedFromSecureWebViewSender.swift
+//  StripeConnect
+//
+//  Created by Chris Mays on 8/7/24.
+//
+
+import Foundation
+
+struct ReturnedFromSecureWebViewSender: MessageSender {
```

Comment:
- 🚫 **Blocker:** Naming (see comment on OpenSecureWebViewMessenger for context)
- nit: docstrings
```suggestion
/// Notifies that the user finished the flow within the `ASWebAuthenticationSession`
struct ReturnedFromAuthenticatedWebViewSender: MessageSender {
```